modular_skyrat/modules/death_consequences_perk/death_consequences_trauma.dm

```diff
@@ -32,20 +40,12 @@
 	var/formaldehyde_death_degradation_mult = 0
 	/// If our victim is alive and is metabolizing rezadone, we will reduce degradation by this amount every second.
 	var/rezadone_degradation_decrease = DEATH_CONSEQUENCES_DEFAULT_REZADONE_DEGRADATION_REDUCTION
-	/// Only rezadone at or above this purity can reduce degradation.
-	var/rezadone_minimum_purity = 100
-
-	/// When strange reagent is metabolized, degradation is reduced by this.
-	var/strange_reagent_degradation_decrease = DEATH_CONSEQUENCES_DEFAULT_STRANGE_REAGENT_DEGRADATION_REDUCTION
 
 	/// When eigenstasium is metabolized, degradation is reduced by this.
 	var/eigenstasium_degradation_decrease = DEATH_CONSEQUENCES_DEFAULT_EIGENSTASIUM_DEGRADATION_REDUCTION
 
-	/// When sansufentanyl is metabolized, degradation is reduced by this.
-	var/sansufentanyl_degradation_decrease = DEATH_CONSEQUENCES_DEFAULT_SANSUFENTANYL_DEGRADATION_REDUCTION
-
 	/// If our victim is dead, their passive degradation will be multiplied against this if they are in stasis.
-	var/on_stasis_death_degradation_mult = 0
+	var/on_stasis_passive_degradation_mult = 0
```

Comment:
I think this makes a bit more sense as a variable name, and it doesn't have an acronym in it. Keep in mind that acronyms are usually pretty bad for others to understand your code quickly, even if just by a fraction of a second. It adds up to the complexity very quickly.
```suggestion
	var/stasis_passive_degradation_multiplier = 0
```